docs/docs/50-roadmap.md

```diff
@@ -33,22 +33,34 @@ __Status:__ Completed
 
 ## v0.4.0
 
-__Status:__ In Progress
+__Status:__ Completed
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| `Warehouse` Rules/Filters | feature | Introduce features to constrain the conditions under which new `Freight` is produced. |
-| Project Management | feature | <ul><li>Introduce `Project` CRD to simplify onboarding and project lifecycle management.</li><li>Aggregate important status information at the `Project` level.</li><li>Introduce additional `PromotionPolicy` options.</li><li>Credential management via CLI and UI.</li></ul> |
+| `Warehouse` Rules/Filters | feature | Introduced optional tag-based constraints on Git repository subscriptions. |
+| Project Management | feature | <ul><li>Introduced `Project` CRD to simplify project initialization.</li><li>Removed `PromotionPolicy` CRD and folded its functionality directly into the `Project` CRD.</li></ul> |
 
 ## v0.5.0
 
+__Status:__ In Progress
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| Project Management | feature | Add sensible `ServiceAccount`s, `Role`s, and `RoleBinding`s to boilerplate project setup. |
+| `Warehouse` Rules/Filters | feature | <ul><li>Introduce optional path-based constraints on Git repository subscriptions.</li><li>Introduce optional windows wherein production of new `Freight` is allowed or prohibited.</li><li>Introduce option to pause or "freeze" `Freight` production.</li></ul> |
+| `Stage` Rules/Filters | feature | <ul><li>Introduce optional windows wherein promotions to a `Stage` are allowed or prohibited.</li><li>Introduce option to pause or "freeze" promotions to a `Stage`.</li></ul> |
```

Comment:
I'd like to defer the windowing/freezing features for now, since I'd like to focus on more longer-term directional improvements (patch promotions + extensible promotions).

Reply:
Works for me.